UI/obs-app.cpp

```diff
@@ -69,6 +69,25 @@
 #include "ui-config.h"
 
 using namespace std;
+long alloc_count;
+
+void *operator new(size_t size)
+{
+	alloc_count++;
+	return malloc(size);
+}
```

Comment:
This violates `new` semantics, if allocation fails a `std::bad_alloc` must be thrown.

Reply:
Added the std::bad_alloc exception.